rust/frontend/src/server.rs

```diff
@@ -254,25 +271,50 @@ impl FrontendServer {
 // These handlers simply proxy the call and the relevant inputs into
 // the appropriate method on the `FrontendServer` struct.
 
+/// Health check endpoint that returns 200 if the server and executor are ready
+#[utoipa::path(
+    get,
+    path = "/api/v2/healthcheck",
```

Comment:
curious if it's possible for utoipa to infer this so we don't have to define the method and path twice?